<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_login">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>40</y>
        <width>761</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton, #pushButton_2, #pushButton_3, #pushButton_4, #pushButton_5{
background-color: rgba(0,0,0,0);
color: rgb(195, 7, 63);
}</string>
      </property>
      <widget class="QLabel" name="label_left_bg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>331</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(:/pictures/pictures/test2.png);
border-top-left-radius: 50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_right_bg">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>381</width>
         <height>581</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image:url(:/pictures/pictures/rightBG2.png);
border-bottom-right-radius: 50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="login_label">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>110</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 7, 63);</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_input">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>200</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(195,7,63);
color:rgb(255,255,255);</string>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_input">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>290</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(195,7,63);
color:rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="login_btn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>410</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(211, 211, 211, 255), stop:1 rgba(229, 9, 20, 255));
  color: rgba(211, 211, 211, 255);
  border-radius:14px;
}
QPushButton:hover{
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(68, 176, 158, 255), stop:1 rgba(26, 26, 29, 255));
color: rgba(68, 176, 158, 255);
}
QPushButton:pressed{
  padding-left:5px;
  padding-top:5px;
  background-color: rgba(68, 176, 158, 255);
}</string>
       </property>
       <property name="text">
        <string>E n t e r</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QLabel" name="forget_label">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>480</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color:rgb(0, 170, 255);
}
QLabel:hover{
color:rgb(0, 85, 255);
}</string>
       </property>
       <property name="text">
        <string>Забыли логин или пароль?</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>510</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="discord_btn">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
background-color: rgba(0,0,0,0);
color:rgb(173, 173, 173);
}</string>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="instagram_btn">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(131, 58, 180, 255), stop:1 rgba(255, 220, 128, 255));
background-color: rgba(0,0,0,0);
}</string>
          </property>
          <property name="text">
           <string>Q</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skype_btn">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
color:rgb(85, 170, 255);
background-color: rgba(0,0,0,0);
}</string>
          </property>
          <property name="text">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="twitter_btn">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
color:rgb(85, 170, 255);
background-color: rgba(0,0,0,0);
}</string>
          </property>
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="register_btn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>370</y>
         <width>91</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover{
color: rgba(68, 176, 158, 255);
}
QPushButton{
background-color: rgba(0,0,0,0);
color: rgb(195, 7, 63);
}</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exit_btn">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>40</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgba(0,0,0,0);
}
QPushButton:hover{
background-color: rgb(195, 7, 63);
border:none;
border-radius: 20px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/pictures/pictures/icon_close.png</normaloff>:/pictures/pictures/icon_close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_signin">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>40</y>
        <width>761</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton, #pushButton_2, #pushButton_3, #pushButton_4, #pushButton_5{
background-color: rgba(0,0,0,0);
color: rgb(195, 7, 63);
}</string>
      </property>
      <widget class="QLabel" name="label_left_bg_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>381</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(:/pictures/pictures/signLeftBG.png);
border-top-left-radius: 50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_right_bg_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>331</width>
         <height>581</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/pictures/pictures/signRightBG.png);
border-bottom-right-radius: 50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="signup_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 7, 63);</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_input_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(195,7,63);
color:rgb(255,255,255);</string>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_input_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>280</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(195,7,63);
color:rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signup_btn">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>440</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(229, 9, 20, 255), stop:1 rgba(211,211, 211, 255));
  color: rgba(211, 211, 211, 255);
  border-radius:14px;
}
QPushButton:hover{
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(26, 26, 29, 255), stop:1 rgba(68, 176, 158, 255) );
color: rgba(68, 176, 158, 255);
}
QPushButton:pressed{
  padding-left:5px;
  padding-top:5px;
  background-color: rgba(68, 176, 158, 255);
}</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>510</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="discord_btn_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
background-color: rgba(0,0,0,0);
color:rgb(173, 173, 173);
}</string>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="instagram_btn_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(131, 58, 180, 255), stop:1 rgba(255, 220, 128, 255));
background-color: rgba(0,0,0,0);
}</string>
          </property>
          <property name="text">
           <string>Q</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skype_btn_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
color:rgb(85, 170, 255);
background-color: rgba(0,0,0,0);
}</string>
          </property>
          <property name="text">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="twitter_btn_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Social Media Circled</family>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover{
color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 166, 255, 255));
}
QPushButton{
color:rgb(85, 170, 255);
background-color: rgba(0,0,0,0);
}</string>
          </property>
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="confirmpass_input">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>360</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(195,7,63);
color:rgb(255,255,255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exit_btn_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgba(0,0,0,0);
}
QPushButton:hover{
background-color: rgb(195, 7, 63);
border:none;
border-radius: 20px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/pictures/pictures/arrow-left.png</normaloff>:/pictures/pictures/arrow-left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>75</width>
         <height>75</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
